### PR TITLE
Inital version of admin-panel

### PR DESCRIPTION
**Pull request by martijnjanssen**
This is an initial version of the admin panel. Some things are broken right now, can't figure out why:
- [ ] 404 fallback is broken for some reason
- [ ] Alignment of content is only fixed after a resize
- [ ] Build succeeding even though fragments are missing
- [x] Linting warning `'Polymer' is not defined`
- [x] Linting warning `Use of the function form of "use strict"`

I haven't really looked at these issues yet though.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,20 +1,122 @@
...
+        onload();
+      }
+    })();
+  </script>
 
-    <title>lancie-admin</title>
-    <meta name="description" content="lancie-admin description">
+  <script>
+  'use strict';
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (TimvdLippe) [inline index.html]: Analytics isn't really needed in the admin panel right?

**Review by TimvdLippe — COMMENTED** [review]
Add the linter configuration: https://github.com/AreaFiftyLAN/lancie-frontend/blob/master/.jshintrc#L18

**Comment by TimvdLippe** [comment]
> Build succeeding even though fragments are missing

Requires my script, #TODO

File: src/lancie-admin-layout/lancie-admin-layout.html [hunk 2]
```diff
@@ -95,6 +114,13 @@
         }
       },
 
+      attached: function() {
+        this.async(function() {
+          this.$.drawerLayout.resetLayout();
+          this.$.headerLayout.resetLayout();
```
Comment (TimvdLippe) [inline src/lancie-admin-layout/lancie-admin-layout.html:121]: Iewl :sob:

Comment (martijnjanssen) [reply src/lancie-admin-layout/lancie-admin-layout.html:121]: Yeah, this is the quick and dirty version of doing this. Don't know how to fix the layout thingy...

**Comment by MatthijsKok** [comment]
Looking good! Only had a shallow look at the code though.

**Comment by MatthijsKok** [comment]
I just tried resolving these issues, to no avail. If no one else wants to try this, i'm merging this. I know it's not perfect, but we want an MVP out as soon as possible. Call it Agile if you want to :^)

**Comment by martijnjanssen** [comment]
Do create issues for them if you do merge.

**Pull request merged**